### DISCUSSION TITLE
What does passing `stale: undefined` to the `cacheLife` function mean?

### DISCUSSION THREAD

**Discussion by tomdohnal** [vercel/next.js#71897] [App Router]
Hey everyone,

I was wondering if specifying `stale: undefined` to the `cacheLife` function mean.

In the docs, it says that the `stale` property specifies: "Duration the client should cache a value without checking the server."

So does passing `stale: undefined` mean:
1. the client will _never_ check the server
2. or the client will _always_ check the server

Thank you!

**Comment by aniruddhaadak80** [comment]
Hi @tomdohnal ,

Passing `stale: undefined` to the `cacheLife` function means that the client will **always check the server** for the latest data. When `stale` is set to `undefined`, it implies that there is no duration defined for the cached value, resulting in the client bypassing the cache and fetching fresh data from the server every time.

Thanks!
